.travis.yml

```diff
@@ -26,13 +26,15 @@ before_script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       pip install git+https://github.com/SUPERAndroidAnalyzer/travis-cargo.git@fix-coveralls --user &&
-      export PATH=$HOME/.local/bin:$PATH
+      export PATH=$HOME/.local/bin:$PATH &&
+      (travis-cargo install rustfmt || true)
       fi
 
 # The main build
 script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      travis-cargo --only stable fmt -- -- --write-mode=diff &&
```

Comment:
This makes the build fail. Maybe we can use `cargo` itself, since it's installed in container builds. But try testing it without the second `--`. If it doesn't work, I would simply use `cargo fmt -- --write-mode=diff`.